fastly.toml

```
@@ -1,5 +1,5 @@
-manifest_version = 2
-name = "Default starter for JavaScript"
-description = "A basic starter kit that demonstrates routing, simple synthetic responses and overriding caching rules."
 authors = ["<oss@fastly.com>"]
+description = "A basic starter kit that demonstrates routing, simple synthetic responses and overriding caching rules."
 language = "javascript"
+manifest_version = 2
+name = "Default starter for JavaScript"
```

```suggestion
manifest_version = 2
name = "Default starter for JavaScript"
description = "A basic starter kit that demonstrates routing, simple synthetic responses and overriding caching rules."
authors = ["<oss@fastly.com>"]
language = "javascript"
```

These changes are unnecessary.